### ISSUE TITLE
[Confused] Why does updateLocalTransform of  Transform  not compute  lt.tx & lt.ty ?

### ISSUE THREAD

**Issue by finscn** [pixijs/pixi.js#3369]
in updateTransform of Transform : 
```js
    updateTransform(parentTransform)
    {
        const pt = parentTransform.worldTransform;
        const wt = this.worldTransform;
        const lt = this.localTransform;

        const a  =  this._cr * this.scale.x;
        const b  =  this._sr * this.scale.x;
        const c  = -this._sr * this.scale.y;
        const d  =  this._cr * this.scale.y;

        lt.a = (this._cy * a) + (this._sy * c);
        lt.b = (this._cy * b) + (this._sy * d);
        lt.c = (this._nsx * a) + (this._cx * c);
        lt.d = (this._nsx * b) + (this._cx * d);

//  ************************************
//  ******* compute lt.tx & lt.ty *********
//  **
        lt.tx = this.position.x - ((this.pivot.x * lt.a) + (this.pivot.y * lt.c));
        lt.ty = this.position.y - ((this.pivot.x * lt.b) + (this.pivot.y * lt.d));
//  **
//   ************************************

        // concat the parent matrix with the objects transform.
        wt.a = (lt.a * pt.a) + (lt.b * pt.c);
        wt.b = (lt.a * pt.b) + (lt.b * pt.d);
        wt.c = (lt.c * pt.a) + (lt.d * pt.c);
        wt.d = (lt.c * pt.b) + (lt.d * pt.d);
        wt.tx = (lt.tx * pt.a) + (lt.ty * pt.c) + pt.tx;
        wt.ty = (lt.tx * pt.b) + (lt.ty * pt.d) + pt.ty;

        this._worldID ++;
    }
```

but  in  updateLocalTransform of Transform : 
```
    updateLocalTransform()
    {
        const lt = this.localTransform;
        const a  =  this._cr * this.scale.x;
        const b  =  this._sr * this.scale.x;
        const c  = -this._sr * this.scale.y;
        const d  =  this._cr * this.scale.y;

        lt.a = (this._cy * a) + (this._sy * c);
        lt.b = (this._cy * b) + (this._sy * d);
        lt.c = (this._nsx * a) + (this._cx * c);
        lt.d = (this._nsx * b) + (this._cx * d);

        // *******  where are  lt.tx & lt.ty ?

    }
```

If this is not a bug , why don't compute tx & ty ?

Thanks.

 I'm reading & learning  the source code of PIXI.js .
 there are some things make me confused. 
So , from now on , I'll add a tag [confused] for this kind issues .

**Comment by staff0rd** [comment]
Likely a question for @ivanpopelyshev

**Comment by ivanpopelyshev** [comment]
This function isn't used yet, and I fixed that in latest PR, #3382

**Comment by finscn** [comment]
Could you  also see this https://github.com/pixijs/pixi.js/issues/3370 ，please  ？
